### ISSUE TITLE
Revise and update documentation

### ISSUE THREAD

**Issue by PTB-OEDA** [openeventdata/UniversalPetrarch#25]
Build off the earlier documentation to provide a better manual for how to use the code.

**Comment by ahalterman** [comment]
Yes, distinct from the white paper in #51 would be a "vignette" or how-to document if we want people to use UP.
